src/parser/jobs/mnk/modules/RiddleOfFire.tsx

```diff
@@ -82,7 +82,7 @@ export class RiddleOfFire extends Analyser {
 	@dependency private timeline!: Timeline
 
 	private history: Riddle[] = []
-	private riddle?: Riddle
+	private riddle: Riddle | undefined
```

Comment:
See also comment on Forms

Reply:
Same deal as above although I guess technically using RoF is optional, even if not using it is a terrible idea.

Reply:
Making this one optional afterall.